perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -148,6 +147,9 @@ def apply_action(action_name, comment, descriptions):
   else:
     raise ValueError('Action "%s" not yet implemented.' % self.action_name)
 
+def timestamp_to_datetime(timestamp):
```

Comment:
The '..to_datetime' part is a little confusing, because a datetime is also a python type (i.e., what `datetime.now()` returns), whereas this is returning a string.

Maybe rename to "timestamp_string"?

Reply:
Done.